src/main/java/cucumber/pageObjects/Bureau.java

```diff
@@ -142,5 +142,9 @@ public void pressBackLink() throws Exception {
 			}
 		}
 	}
+	public void clickChangeLink(){
+		WebElement changeLink = driver.findElement(By.xpath("//*[contains(text(), 'Change')][not(@href='/auth/courts-list')]"));
```

Comment:
Why does this class not use @FindBy annotations like the other page objects?

Reply:
I'll change this, I think initially I was intending on doing something different but changed my mind